SECURITY.md

```diff
@@ -0,0 +1,123 @@
+
+Release artifacts are signed via
+[sigstore](https://www.sigstore.dev). See
+[release-sign.yml](.github/workflows/release-sign.yml) for details.
+
+To verify a downloaded release at a given tag:
+
+    % pip install sigstore
+    % sigstore verify github --cert-identity https://github.com/AcademySoftwareFoundation/rez/.github/workflows/release-sign.yml@refs/tags/<tag> rez-<tag>.tar.gz
+
+## Security Expectations
+
+### Software Features
+
+- The rez project implements a package management solution that is agnostic
+  to build system, shell, platform, architecture, operating system, or
+  packaged toolset. rez can be used to package python, javascript, C++, or
+  even binaries. As long as it is possible to express the way your package
+  modifies the environment to expose itself to be consumed, rez can package it.
+
+- rez is implemented in python, and consists primarily of simply its own
+  source, a handful of vendored python packages, and is installed to a system
+  in the form of a virtualenv-powered executable, tied to a python interpreter.
+
+- rez exposes a robust CLI which can be used to do common operations like build
+  or release packages, to resolve and drop into a shell for a given environment,
+  or to freeze context files that can be used to bake and reuse resolves.
+
+- rez exposes an API that can be used, in and out of isolation, to perform many
+  of the same tasks, as well as more granular operations, so that groups can
+  also leverage many of the low-level constructs that make the higher-level CLI
+  usages possible. An important disclaimer is that the rez API has not yet been
+  robustly reviewed and modified to express what the boundaries of the public vs
+  private API are, meaning that some internals may currently be exposed that rez
+  may in the future be moved, hidden, or removed.
+  
+- rez reads and writes to/from stdout, as well as to/from json-like rxt files.
+  rez also writes temporary shell context files like .sh or .bat scripts on a
+  shell-by-shell basis, as a shell entrypoint into the chosen environment.
+
+- rez exposes a rich plugin system which can be used and configured in order to
+  augment and expand its use-cases or integrations per the users preferences.
+
+- rez will engage in network calls only if configured to do so. Typically, this
+  would be the case with the built-in `memcached` functionality, or to publish
+  `amqp` messages on package release.
+
+- rez does not handle, or expect to handle, any sort of login credentials to any
+  network, file system, or verson-control system. rez expects that these details
+  are handled independently and outside of rez.
+
+- rez packages can, by definition and by design, result in the execution of
+  arbitrary code. It is critical that users of rez maintain their own strict
+  control over their own package repositories, and not trust arbitrary packages
+  given to them by unknown sources. rez takes no responsibility for malicious
```

Comment:
The responsibility clause might already be part of the license file?

Reply:
extended this verbiage to plugins and configs